### ISSUE TITLE
400 error when editing metadata

### ISSUE THREAD

**Issue by scare376** [Kareadita/Kavita#1191]
**Describe the bug**
I'm getting a 400 error when editing metadata. The metadata saves but only when leaving the page.

**To Reproduce**
1. Change metadata with the edit series information button on series page
2. Click save button (400 error will appear)
3. close metadata window with x or close button
4. check edit series information. changes were not saved.

**Expected behavior**
Fields edited keep the changes.

**Screenshots**
https://user-images.githubusercontent.com/2068470/161441292-159bdb24-73c6-454c-a29a-00ede0995b6d.mp4

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Opera GX, Google Chrome
 - Version: 0.5.2.1

**Additional context**
After hitting the save button instead of checking if the information saved if you leave the page and open the series back up the changes are applied.

**Comment by majora2007** [comment]
Issue:
```
2022-04-03T13:50:49.7121316-04:00	FAIL	[API.Services.SeriesService]	[0]	There was an exception when updating metadataSystem.InvalidOperationException: Sequence contains more than one matching element
   at System.Linq.ThrowHelper.ThrowMoreThanOneMatchException()
   at System.Linq.Enumerable.TryGetSingle[TSource](IEnumerable`1 source, Func`2 predicate, Boolean& found)
   at API.Services.SeriesService.UpdatePeopleList(PersonRole role, ICollection`1 tags, Series series, IReadOnlyCollection`1 allTags, Action`1 handleAdd, Action onModified) in /home/runner/work/Kavita/Kavita/API/Services/SeriesService.cs:line 319
   at API.Services.SeriesService.UpdateSeriesMetadata(UpdateSeriesMetadataDto updateSeriesMetadataDto) in /home/runner/work/Kavita/Kavita/API/Services/SeriesService.cs:line 112
```

**Comment by majora2007** [comment]
@scare376 are you able to test if this is still an issue on the nightly (v0.5.1.2)? I think I had put a solution in there for something related to this.

**Comment by majora2007** [comment]
As discussed, this seems to be fixed by Sort Name fix: #1190
